apps/src/code-studio/pd/application/teacher/ImplementationPlan.jsx

```diff
@@ -126,6 +81,15 @@ const ImplementationPlan = props => {
           />
         </>
       );
+    } else if (!!data.willTeach) {
+      return (
+        <p style={styles.error}>
+          Note: This program is designed to work best for teachers who are
```

Comment:
I imagine we don't have plans to translate the application?

Reply:
We don't translate the application because the workshops are not offered in other languages currently.